src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Base/MyExtensionsApp._1.Base.csproj

```diff
@@ -2,10 +2,11 @@
 	<PropertyGroup>
 		<!-- NOTE: The TargetFramework is required by MSBuild but not used as this project is not built. -->
 		<TargetFramework>$libraryBaseTargetFramework$</TargetFramework>
+		<EnableDefaultItems>false</EnableDefaultItems>
 	</PropertyGroup>
 
 	<ItemGroup>
-		<!-- NOTE: The AppHead is accessible in the Platform heads -->
-		<None Remove="AppHead.*" />
+		<None Include="**\*" Exclude="obj\**;bin\**;*.csproj" />
+		<None Include="AppHead.xaml.cs" DependentUpon="AppHead.xaml" />
```

Comment:
Should this be Update? Since this file will already be included in previous line